### ISSUE TITLE
Config for penalties to only be applied when player has shank modifier

### ISSUE THREAD

**Issue by yeelp** [yeelp/Scaling-Feast#116]
Config option to only apply penalties if it leaves the shank modifier >= 0

**Comment by James103** [comment]
Also if the proposed config option is enabled, and you get a -5 penalty to max hunger, while you have only eaten 1 shank (+2 max hunger), it sets your shank modifier to zero but the rest of the penalty is not applied.

**Comment by yeelp** [comment]
Absolutely, that was what I had in mind, I just did a poor job explaining that in the original description. Basically, bound all penalties and the shank bonus modifiers below at 0.
